docs/assembly-control-unit.md

```diff
@@ -1,164 +1,274 @@
 
-### Verify stepper motor wires and tune the potentiometers
+<IMG ALT="Potentiometer" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="600" />
 
-You can do this step before assembling the Reactor if you have:
+<IMG ALT="Position of potentiometer" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="300" />
 
-- A working Pi,
-- a working Arduino with the motor shield connected to a 12v power source,
-- stepper motors or stepper-driven peristaltic pumps, and
-- the stepper motor cables.
+Note:
+In the image above, the potentiometers are circled and their slightly-flat sides are pointed to the left. This is the "slightly-open" position.
 
-You can also perform this step after assembling the other units. You **DEFINITELY** want to tune the potentiometers. If you don't, the stepper motors will likely behave very strangely and could be damaged by excess voltage. You can use the Test Recipe on the MicroLab to do this, which has an option to manually run each of the reagent pumps, as well as activing the stirrer and heating tools. You can also do this after assembling the other units, but be sure to do it before you close the lid on the Control Unit.
+**WARNING!** Failure to correctly tune the potentometers can damage the motor controller and the motors themselves.
 
-#### Motor board potentiometer tuning
+How to tune:
+You will **gently** turn the potentometers on the motor control boards clockwise or counter-clockwise using a small phillips head screw driver. Be careful to stop once there is any resistance, do not to turn past the resistance.
+* Turning them clockwise "closes" the voltage down
+* Turning them counter-clockwise "opens" the voltage up
 
-<IMG ALT="Potentometers" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="600" />
+1. Attach the Arduino's stepper motor cables to the stepper motor on the peristalic pump.
+<IMG ALT="Attaching pump cables" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="300" />
+1. Double check the everything is connected properly.
+1. Start with the potentiometers in the "closed" position.
+<IMG ALT="Close up of closed potentiometer" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="300" />
+1. Power on the Raspberry Pi, the Arduino shield and the pumps.
+1. **Don't we need to add cable between Arduino and Pi?**
+1. On the Raspberry Pi, the MicroLab software will load, locate the Test Recipe and choose to manually run each of the peristalic pumps. **More detail needed**<br>
+<IMG ALT="Screencap of pump on" SRC="./media/control-unit/potentiometers-annotated.jpg" WIDTH="300" />
+1. Slowly turn the potentiometer counter-clockwise an eighth of a turn at a time to "open" them. Check if the stepper motors turn smoothly. If they do not move, "open" the potentiometer an eighth-turn and try again. If they move erratically, your voltage is likely too high and you "close" the potentiometer.
```

Comment:
"If they move erratically, your voltage is likely too high and you "close" the potentiometer." ->

"If they move erratically, the voltage is likely too high. In this case, turn the potentiometer slightly clockwise to "close" it."